README.md

```diff
@@ -1,48 +1,97 @@
 # test_creation
 
-To be filled
+A package for creating and managing software tests for applied machine learning code.
+
+![](img/logo.jpg)
```

Comment:
Can use a higher resolution picture and control the size 
```suggestion
<figure>
    <img src="./img/logo.png?raw=true" width="175" height="175">
</figure>
```
![fixml_logo](https://github.com/UBC-MDS/fixml/assets/38127942/c514463a-1d53-46d4-898f-390ea633c72a)

Reply:
Done